### ISSUE TITLE
Allow users to customise the colors of FrameTimeGraph

### ISSUE THREAD

**Issue by Freyja-moth** [bevyengine/bevy#21305]
## What problem does this solve or what need does it fill?

Some user with colour blindness (specifically red-green) may find it difficult to differentiate between the different states of the frame time graph.

## What solution would you like?

Either allow uses to specify the colours that the frame time graph uses, or add an enum with common colourblindness types and adjust colours based on variant.

If we go with the second option we should consider making it generic for all widgets and so on.

## What alternative(s) have you considered?

Not too big a deal as the height of the graph can still be used, but I still feel we should do this anyways.

**Comment by afonsolage** [comment]
Additional context, currently the colors are hard coded on shader:

https://github.com/bevyengine/bevy/blob/eac8bc7cc4b35c98db00a6cf1fd39377afcbac61/crates/bevy_dev_tools/src/frame_time_graph/frame_time_graph.wgsl#L13-L20

This should be simple to do, just add something like `low_color` and `high_color` to `FrameTimeGraphConfig` and pass it to shader.

Related: https://github.com/bevyengine/bevy/issues/21311

**Comment by alice-i-cecile** [comment]
Strongly in favor of this :)
